src/battle_util.c

```diff
@@ -6621,14 +6622,18 @@ static u8 TryConsumeMirrorHerb(u32 battler, bool32 execute)
 {
     u8 effect = 0;
 
-    if (gProtectStructs[battler].eatMirrorHerb) {
+    if (gProtectStructs[battler].eatMirrorHerb)
+    {
         gLastUsedItem = gBattleMons[battler].item;
         gBattleScripting.savedBattler = gBattlerAttacker;
         gBattleScripting.battler = gBattlerAttacker = battler;
         gProtectStructs[battler].eatMirrorHerb = 0;
-        if (execute) {
+        if (execute)
+        {
             BattleScriptExecute(BattleScript_MirrorHerbCopyStatChangeEnd2);
-        } else {
+        }
```

Comment:
More of a question than an actual review, but I believe it's inconsistent what happens to conditions (both if and else) when there's only one function that gets called if the condition is met. I've seen braces around it, I've also just seen a simple indentation. Probably something to sort out (possibly pret-side) and make more consistent, too

Reply:
This is the logic I followed here:
- `else` has 2 lines, therefore braces are needed.
- If/else chain has at least 1 block with required braces, therefore all of them should have braces.

I could PR some of these style adjustments, as I've already PR'd a matching PR to pret for braces.
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/1554c1f4-ec7d-4003-8b16-40e812245de2) 
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/cc650361-6a68-4140-82de-10dc54d9849e)

Reply:
~~Let's start a war in pret and move over the objectively correct style lol~~
```
if (condition) {
    // code
}
```

Reply:
~~pokeheartgold does it lol~~